static/app/views/releases/list/index.tsx

```diff
@@ -575,6 +592,12 @@ const AlertText = styled('div')`
   }
 `;
 
+const PageFilterWrapper = styled('div')`
```

Comment:
I might do similar to what I did here https://github.com/getsentry/sentry/pull/33034/files#diff-af8c8438bd2c80266943452229b652c5b70c1ced47ee0135b5dd7bd53ffb8824R755-R758

Reply:
ahh I mainly did it this way to avoid it overflowing the page if the selected filters make the bar too big or the window gets too small

![image](https://user-images.githubusercontent.com/9372512/160720961-d1b92d2a-39fc-40ee-91b3-d2946b386784.png)

Reply:
but I can probably also just do `max-width: 100%` to achieve the same thing with `width: max-content`